lib/spreedly/base.ex

```diff
@@ -71,18 +71,18 @@ defmodule Spreedly.Base do
     {:error, extract_reason(body)}
   end
 
-  defp reason_phrase(integer) do
+  defp reason_phrase(status_code) do
     statuses = %{
-      408 => "Request timeout",
-      429 => "Too many requests, rate limit exceeded",
+      408 => "Request Timeout See https://docs.spreedly.com/reference/api/v1/#408-request-timeout",
+      429 => "Too many requests, rate limit exceeded. See https://docs.spreedly.com/reference/api/v1/#429-too-many-requests",
       500 => "Internal Server Error",
       501 => "Not Implemented",
       502 => "Bad Gateway",
-      503 => "Service Unavailable - Spreedly is temporarily unable to process the request.",
+      503 => "Service Unavailable. See https://docs.spreedly.com/reference/api/v1/#429-too-many-requests",
```

Comment:
Link should be https://docs.spreedly.com/reference/api/v1/#503-service-unavailable